### PR TITLE
doc: Add Thread and Zigbee to the nRF53 user guide

### PR DESCRIPTION
**Pull request by MarekPorwisz**
Mentioned that Thread and Zigbee samples can be run on nRF5340 as well.

Signed-off-by: Marek Porwisz <marek.porwisz@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_nrf5340.rst [hunk 1]
```diff
@@ -147,6 +152,9 @@ Most of these samples should run on the nRF5340 DK, but not all have been thorou
 
 Some Bluetooth LE samples require configuration adjustments to the :ref:`zephyr:bluetooth-hci-rpmsg-sample` sample as described in the `Network samples`_ section.
 
+The |NCS| also provides :ref:`Thread <ug_thread>` and :ref:`Zigbee <ug_zigbee>` samples that can be run on the nRF5340 DK.
```
Comment (wbober) [inline doc/nrf/ug_nrf5340.rst:155]: Do we need to say that the child image is built & flashed automatically.

**Review by wbober — APPROVED** [review]

File: doc/nrf/ug_nrf5340.rst (hunk 1 above)
Comment (greg-fer) [reply doc/nrf/ug_nrf5340.rst:156]: @grochu you were planning to rephrase the sentence at the start of the page. Can we maybe rework that sentence and this one in this PR?

**Review by hubertmis — APPROVED** [review]

**Review by edmont — APPROVED** [review]

File: doc/nrf/ug_nrf5340.rst (hunk 1 above)
Comment (grochu) [reply doc/nrf/ug_nrf5340.rst:162]: In the _Application Samples_ and _Network Samples_ sections wouldn't it be cleaner to only mention the specific applications available for these cores respectively, and only describe the automatic building of the child image in the _Building and Programming a sample_ section as it is stated for BLE?

Comment (grochu) [reply doc/nrf/ug_nrf5340.rst:162]: I mean also line 152 could be moved (and maybe reworded) to _"Building and programming a sample"_

**Comment by carlescufi** [comment]
@greg-fer can this be merged?

**Review by greg-fer — APPROVED** [review]

**Comment by greg-fer** [comment]
@carlescufi , it can be merged now, once the CI becomes green.

File: doc/nrf/ug_nrf5340.rst [hunk 2]
```diff
@@ -174,9 +189,12 @@ In a multi-image build, the image for the application core is the parent image,
 For this to work, the network core image must be explicitly added as a child image to one of the application core images.
 See :ref:`ug_multi_image_defining` for details.
 
-The network sample :ref:`zephyr:bluetooth-hci-rpmsg-sample` is automatically added to all Bluetooth Low Energy samples in the |NCS|.
+The network sample :ref:`zephyr:bluetooth-hci-rpmsg-sample` is automatically added to all Bluetooth Low Energy samples in the |NCS|, and the network sample :ref:`nrf-ieee802154-rpmsg-sample` is automatically added to all Thread and Zigbee samples.
```
Comment (hubertmis) [inline doc/nrf/ug_nrf5340.rst:192]:
```suggestion
The network sample :ref:`zephyr:bluetooth-hci-rpmsg-sample` is automatically added to all Bluetooth Low Energy samples in the |NCS|.
```
I would remove 802.15.4, Thead, and Zigbee part from this paragraph. It makes line 193 harder to understand. 802.15.4, Thread, and Zigbee and correctly described in the next paragraph.

**Review by hubertmis — CHANGES_REQUESTED** [review]

File: doc/nrf/ug_nrf5340.rst (hunk 2 above)
Comment (greg-fer) [reply doc/nrf/ug_nrf5340.rst:192]: My bad, will fix asap.

**Review by hubertmis — APPROVED** [review]

**Review by grochu — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]
